cmd/sync/errormessages.go

```diff
@@ -8,3 +8,7 @@ const upstreamNameErrorMsg = "The mandatory field name is either empty or missin
 const upstreamErrorMsgFormat = "The mandatory field %v is either empty or missing for the upstream %v in the config file"
 const upstreamPortErrorMsgFormat = "The mandatory field port is either zero or missing for the upstream %v in the config file"
 const upstreamKindErrorMsgFormat = "The mandatory field kind is either not equal to http or tcp or missing for the upstream %v in the config file"
+const upstreamMaxConnsErrorMsg = "The field max_conns has invalid value %v, must be positive or zero in the config file"
+const upstreamMaxFailsErrorMsg = "The field max_fails has invalid value %v, must be positive or zero in the config file"
+const upstreamFailTimeoutErrorMsg = "The field fail_timeout has invalid value %v"
```

Comment:
Since we are not using the `err` in ParseTime, and if you are not going to use it, maybe we can add here:
```go
const upstreamFailTimeoutErrorMsg = "The field fail_timeout has invalid value %v, must be a valid time string"
```

What do you think? (Same for `upstreamSlowStartErrorMsg`)